public/stylesheet/pumpio.css

```diff
@@ -35,6 +35,10 @@ body {
   width: auto;
 }
 
+.navbar .btn-navbar span {
```

Comment:
This would be more clear if it targeted `.fa.fa-bars`

Reply:
I think specifics styles are harder to maintain and adds unnecessary complexity, the result is the same.

If in the future change de icon name no need changes styles only change the class or if changes the icon family only need to changes classes

https://stackoverflow.com/questions/2281766/generic-vs-specific-element-styles-for-maintainability

Reply:
I'd be okay with that if you add a comment above saying what this applies to. Otherwise future readers may have the same question I did ("what does this even apply to?")

Reply:
Made